README.md

```diff
@@ -72,6 +74,10 @@ Where:
 Enable open loop mode. When enabled, the benchmark client will not
 provide backpressure when resource limits are hit.
 
+--h1-connection-reuse-strategy <HOT|FAIR>
```

Comment:
Drive by comment: FAIR doesn't map to least-recently-used in my mind.  Consider other names like:
most-recently-used | least-recently-used
HOT | COLD

Also should this be lowercase or uppercase?  The default below appears in lowercase, but the values here are uppercase.

Reply:
I like lru/mru, and in fact that is what these options where called initially. I can't remember why I changed it to this. Also, lowercasing makes sense, I'll amend that.

Reply:
lru/mru sgtm.